plugins/koterm.koplugin/main.lua

```diff
@@ -495,19 +491,19 @@ Aliases (shortcuts) to frequently used commands can be placed in:
             {
                 text_func = function()
                     return T(_("Buffer size: %1 kB"),
-                        G_reader_settings:readSetting("KOTerm_buffer_size", 16))
+                        G_reader_settings:readSetting("koterm_buffer_size", 16))
                 end,
                 callback = function(touchmenu_instance)
-                    local cur_buffer = G_reader_settings:readSetting("KOTerm_buffer_size")
+                    local cur_buffer = G_reader_settings:readSetting("koterm_buffer_size")
                     local buffer_spin = SpinWidget:new{
                         value = cur_buffer,
                         value_min = 10,
                         value_max = 30,
                         value_hold_step = 2,
                         default_value = 16,
-                        title_text = _("KOTerm buffer size"),
+                        title_text = _("Terminal emulator buffer size"),
```

Comment:
May be show it's a value in kB, in the title or a description text:
`Terminal emulator buffer size (kB)`